src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -669,6 +755,13 @@ protected void setBelowMaxDepthTransactionLimit(int maxAnalyzedTransactions) {
                 "KPWCHICGJZXKE9GSUDXZYUAPLHAKAHYHDXNPHENTERYMMBQOPSQIDENXKLKCEYCPVTZQLEEJVYJZV9BWU";
 
         //Snapshot
+        boolean LOCAL_SNAPSHOTS_ENABLED = true;
+        boolean LOCAL_SNAPSHOTS_PRUNING_ENABLED = false;
+        int LOCAL_SNAPSHOTS_PRUNING_DELAY = 1000;
+        int LOCAL_SNAPSHOTS_INTERVAL_SYNCED = 10;
+        int LOCAL_SNAPSHOTS_INTERVAL_UNSYNCED = 5000;
+        String LOCAL_SNAPSHOTS_BASE_PATH = "mainnet";
```

Comment:
Do we actually need this configuration variable? This should be automatically be adjusted depending to the network you are using. This way we can also always rely on a standard location for local snapshot files.

Reply:
We already allow users to specify a data directory for their IRI nodes, I believe the local snapshots should be just dropped into that same folder; without giving the user this option, which I don't find the use for.

Reply:
the DB/persistable folder structure should be revisited, but as a different issue IMO.

Reply:
Can we create issues for minor things like this so i can change it afterwards? because once we have it merged i can more easily work on changes since i don't have to do them in 10 different branches.

Reply:
Yes, please, create an issue for this.